[iOS] Browse Superviews for NeedsKeyboard (#793)
nventive/Uno: doc/articles/using-uno-ui.md

<true/>
```

> #### Controlling Keyboard on iOS

Reviewer: ```suggestion
#### Controlling the Keyboard on iOS
```

> If a view needs to keep the keyboard open when you tap on it, you can use SetNeedsKeyboard of the Widow.iOS class.

Reviewer: ```suggestion
If a view needs to keep the keyboard opened when tapping on it, use the `Uno.UI.Controls.Window.SetNeedsKeyboard` method.
```

Reviewer 2: Isn't it an attached property? We could give a XAML snippet instead?
